nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/docs/configuration/adding_your_own.md

```diff
@@ -88,7 +88,7 @@ If you intend to run the pipeline offline, nextflow will not be able to automati
 First, pull the image file where you have an internet connection:
 
 ```bash
-singularity pull --name {{ cookiecutter.pipeline_slug }}.img docker://nf-core/{{ cookiecutter.pipeline_name }}
+singularity pull --name {{ cookiecutter.pipeline_slug }}.img docker://{{ cookiecutter.github_repo }}
```

Comment:
.simg as extension - thats the new Singularity 2.4.x + format.

Reply:
like `/path/to/nfcore-tools-[VERSION].simg`? :-)

Reply:
Almost - better to just have `{{ cookiecutter.pipeline_slug }}.simg` so that people can copy and paste the command. Then it will just be created in the current working directory.

Also note that this change is one of the bad merges - it should be `docker://nfcore/{{ cookiecutter.pipeline_name }}` (note that the original had a hyphen, so both are wrong here)

Reply:
Ah! Great, thanks! ^^ 👍